lib/smart_proxy_openscap/content_parser.rb

```diff
@@ -14,7 +14,6 @@ def validate(file_type, scap_file)
         when 'tailoring_file'
           file = ::OpenscapParser::TailoringFile.new(scap_file)
         end
-        errors << msg unless file.valid?
       rescue Nokogiri::XML::SyntaxError => e
         errors << msg
```

Comment:
should we pass the `error` that's being passed to the `rescue` block instead of passing a hardcoded message?

Reply:
I don't think the Nokogiri error message is a good user facing string. We should only inform about the validity of the file I think. Perhaps we should log the traceback to the log though. I added that, heads up, I didn't test due to the lack of time. Please let me know if it doesn't work.